tests/forte/data/extractor/feature_test.py

```diff
@@ -0,0 +1,149 @@
+#  Copyright 2020 The Forte Authors. All Rights Reserved.
+#  #
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  #
+#       http://www.apache.org/licenses/LICENSE-2.0
+#  #
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+import unittest
+
+from typing import List
+
+from forte.data.extractor.feature import Feature
+
+
+class FeatureTest(unittest.TestCase):
```

Comment:
I guess your feature class is designed for arbitrary types. Can you add tests in additional to `int`?

Reply:
Sure.